app/controllers/keyword/index.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 import PaginationMixin from 'cargo/mixins/pagination';
 
+const { computed } = Ember;
```

Comment:
I haven't heard about this syntax, before, but it seems nice! Could you clarify what's going on here?